doc/source/programs/gdal_raster_resize.rst

```diff
@@ -0,0 +1,90 @@
+.. _gdal_raster_resize_subcommand:
+
+================================================================================
+"gdal raster resize" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Resize a raster dataset.
+
```

Comment:
Would it be good to add "without changing the georeferenced extents"? It may feel obvious, but I know that for some users it is not.

Reply:
done